flink-runtime/src/main/java/org/apache/flink/runtime/operators/coordination/OperatorEventDispatcher.java

```diff
@@ -29,6 +29,9 @@ public interface OperatorEventDispatcher {
     /**
      * Register a listener that is notified every time an OperatorEvent is sent from the
      * OperatorCoordinator (of the operator with the given OperatorID) to this subtask.
+     *
+     * <p>The stream operator with the given OperatorID must implement {@link OperatorEventHandler}
```

Comment:
Because our consistency mechanism requires communications in both directions, all operators with the corresponding OC must be registered as handlers.

This is not required for implementations in this PR and may be used in our solution's second phase to ticket FLINK-26029. So I'll remove this doc for now and possibly add it back in the next PR.